removed underscore as a dependency

dydx opened the pull request:
Not that this is really a huge deal, but it was fairly straight forward. Zero dependencies now!

My update does require node >= 6.x.x. If there are requirements for using more legacy versions, I could pretty easily get that going as well, just a few syntax changes.

kauegimenes commented:
@dydx Thanks for the PR 👍 

Travis is showing some errors with Node v4 any ideas?

dydx commented:
As mentioned in the PR, those are just syntactical things (destructuring and arrow syntax). Relatively easy to downgrade. 

And yeah, moving code to 'core/helper.js' can be accomplished.

kauegimenes commented:
@dydx Thanks for the help =) looks good to merge.

dydx commented:
OK. Moved `lib/helpers.js` into `lib/core/helper.js` and removed the object destructuring; e.g. `const { isArray } = require('./helpers.js');`

Travis is happy now!

kauegimenes commented:
@dydx I just published v2.0.7 to npm.

Thanks again! feel free to submit other PR if you have any idea.

kauegimenes commented:
@dydx We could make the lib even smaller by creating the https://github.com/kauegimenes/jsonexport/blob/master/benchmark/data.json in benchmark.js since its a 10mb file.

kauegimenes commented:
Maybe its better to just use https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package to ignore the benchmark folder
